docs/src/quickstartguide.md

```diff
@@ -0,0 +1,45 @@
+# Quick Start Guide
+
+## Pre-requisites
+
+- **Julia**: HolisticElectricityModel is written in Julia. Instructions to install Julia can be found in the [julia website](https://julialang.org/downloads/).  If this is your first time using Julia visit the official [Getting started with Julia](https://julialang.org/learning/).
+
+- **Solvers**: Running HEM requires access to LP and MIP solvers. Because different choices can be made in this regard, we recommend setting up your computational environment based on one of the environments under [runner](https://github.com/nrel-hem/HolisticElectricityModel.jl/tree/main/runner). Currently, it is recommended to use Gurobi for HPC and Xpress for local runs.
```

Comment:
```suggestion
- **Solvers**: Running HEM requires access to LP and MIP solvers. Because different choices can be made in this regard, we recommend setting up your computational environment based on one of the environments under [runner](https://github.com/nrel-hem/HolisticElectricityModel.jl/tree/main/runner). At NREL we recommend using Gurobi for HPC and Xpress for local runs.
```